op-node/node/config.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 type Config struct {
-	L1  L1EndpointSetup
-	L2s L2EndpointsSetup
+	L1 L1EndpointSetup
+	L2 L2EndpointsSetup
```

Comment:
This can now be renamed to L2EndpointSetup.

Reply:
I found one other plural version that I removed as well.